### PR TITLE
Views

### PR DESCRIPTION
**Pull request by ilyassatlassi**
##HI, Reviewer 👋👋👋

In THis PR:
- Implement the design from the sneak peek wireframes

😊👌

### CODE REVIEW

File: app/views/shared/_user.html.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+    <ul class = "users_list">
+      <li><img src=<%= user.photo %>></li>
+      <div class="user_details"> 
+        <li><%= user.name %></li>
```
Comment (franclobo) [inline app/views/shared/_user.html.erb]: - [ ] Well done using partials, please 🙏🏽 add a link to the name that clicking on each user takes you to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png).

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    <%= render "shared/user", user: @user %>
+    
+   <div class="bio">
+    <h2>Bio</h2>
+    <%= @user.bio%>
+    </div>
+    
+    <%@user_posts.each_with_index do |post, index|%>
+    <div class="post">
+    <h2>post <%=index+1%></h2>
```
Comment (franclobo) [inline app/views/users/show.html.erb]: - [ ] Clicking on each post title takes you to their index page, please 🙏🏽 add the link for a better UX.

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+  <%= render "shared/user", user: @user %>
+    
+    <% if @user.posts.empty? %>
+        <p class="not-available">No post available</p>
+    <% end %>
+    <% @user.posts.each do |post| %>
+        <%= render "shared/post", post: post %>
+        <%= render "shared/comment", comments: post.comments%>
```
Comment (franclobo) [inline app/views/posts/index.html.erb]: - [ ] kindly show the 5 most recent comments on the posts index page for a better UI as shown in the wireframe.

```ruby
<% post.recent_comments.each do |comment| %>
  <p><%= comment.author.name %>: <%= comment.text %></p> 
 <% end %>
```

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @ilyassatlassi, 

# ♻️ Required Changes ♻️

so-close-jim-halpert


Good job so far!
You are doing it well 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Git flow ✔️ 
- Well doe using partials ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @ilyassatlassi   

Your project is complete! Great job for making the changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- No linter errors ✔️
- Great use of partials ✔️
- Descriptive PR ✔️
- Correct Git flow ✔️

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**